templates/shared/_cra-banner.html

```diff
@@ -0,0 +1,9 @@
+<section class="p-strip">
+  <div class="row p-notification--information">
+    <div class="p-notification__content">
+      <div class="p-notification__message">Need information about the CRA? Canonical is committed to delivering Cyber Resilience Act (CRA) compliant Ubuntu. To learn more, 
+        <a href="#get-in-touch">contact our sales team.</a>
```

Comment:
Sorry should have seen this. The link should be to a static page like "/security/contact-us" with the class which invokes the modal.

Reply:
The links on both copy docs seem to be "#get-in-touch" attached to the original link. Should I change it to your suggestion with the the static page link instead?

Reply:
You could do but the authors do not really know how the system works.